lldb-eval/parser.cc

```diff
@@ -137,6 +137,21 @@ std::tuple<lldb::BasicType, bool> PickIntegerType(
   return {lldb::eBasicTypeUnsignedLongLong, true};
 }
 
+lldb::BasicType PickCharType(clang::CharLiteralParser& literal) {
+  if (literal.isMultiChar()) {
+    return lldb::eBasicTypeInt;
+  } else if (literal.isAscii()) {
+    return lldb::eBasicTypeChar;
+  } else if (literal.isWide()) {
+    return lldb::eBasicTypeWChar;
+  } else if (literal.isUTF16()) {
+    return lldb::eBasicTypeChar16;
+  } else if (literal.isUTF32()) {
+    return lldb::eBasicTypeChar32;
+  }
+  return lldb::eBasicTypeChar;
```

Comment:
Is there a valid use case when we get here? It looks like all valid options are covered above, so we can return `lldb::eBasicTypeInvalid` instead.